src/components/view/ListView.vue

```diff
@@ -266,14 +266,19 @@ export default {
       api('updateConfiguration', {
         name: record.name,
         value: this.editableValue
-      }).then(() => {
+      }).then(json => {
         this.editableValueKey = null
 
         this.$message.success('Setting Updated: ' + record.name)
-        this.$notification.warning({
-          message: 'Status',
-          description: 'Please restart your management server(s) for your new settings to take effect.'
-        })
+        if (json.updateconfigurationresponse &&
```

Comment:
@shwstppr would this work if API doesnot have the isdynamic in the API response?

Reply:
@rhtyd should be okay with `!json.updateconfigurationresponse.configuration.isdynamic &&`

https://jsfiddle.net/shwstppr/x5undf7a/

Reply:
okay